challengeutils/wiki.py

```diff
@@ -0,0 +1,184 @@
+"""Download and sync wiki tools"""
+import json
+import os
+import typing
+
+import synapseclient
+from synapseclient import Synapse
+
+
+def pull_wiki(syn: Synapse, project: str,
+              workdir: str = "./") -> typing.List[dict]:
+    """Downloads each wikipage's content into a markdown file and
+    stores a configuration file
+
+    Args:
+        syn: Synapse connection
+        project: synapseclient.Project or its id
+        workdir: Location to download markdown files and wiki_config.json
+                 into. Defaults to location of where code is being
+                 executed.
+
+    Returns:
+        Wiki Configuration
+        [
+            {
+                "id": "111",
+                "title": "title",
+                "parentId": "33333",
+                "markdown_path": "markdown.md"
```

Comment:
Replace to more realistic example, e.g. "markdown_path": "home.md".

The "md" extension and description of the function already make it clear that it is a markdown file.